### DISCUSSION TITLE
UI scaling problem in std (in game borders)

### DISCUSSION THREAD

**Discussion by anthonyrepec** [ppy/osu#20880] [Q&A]
### Type

Cosmetic

### Bug description

With certain skins that have borders on them such as owotuna 1.4 Mathi cyberdark, decreasing the UI scaling can unalign the border with the rest of the UI in game so the score, accuracy and whatnot are in place but the border isn't, shown in a video down below. Not caused by windowed, only on because relive wouldn't let me record osu if it was in fullscreen.

### Screenshots or videos

https://user-images.githubusercontent.com/116473781/197372287-834312e5-80ec-4beb-9006-be33fb1187a5.mp4

### Version

2022.1022.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9845720/database.log)
[network.log](https://github.com/ppy/osu/files/9845721/network.log)
[performance.log](https://github.com/ppy/osu/files/9845722/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9845723/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9845724/updater.log)

**Comment by peppy** [comment]
This is due to the skin using very unsupported methods of showing a border. The recommended fix for now is to remove the border from the skin and use the provided border setting in-game.
